### PR TITLE
Updated the  docs to include recent changes in J5

### PR DESCRIPTION
**Pull request by nebrius**
Note: the docs here were based on the PR that @dtex did for Raspi IO (https://github.com/nebrius/raspi-io-core/pull/5).

### CODE REVIEW

**Review by dtex — APPROVED** [review]
Yes, this looks good to me

**Comment by nebrius** [comment]
@rwaldron can you merge this when you get a chance? (assuming of course you don't have comments)

File: README.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ Data writing operations must be executed in order of instruction.
 
 #### servoWrite(pin, value)
 
-- Accept a `value` in degrees (0-180) that is written to the specified `pin`.
+- Accept a `value` that is written to the specified `pin`. If the value is between 0 and 180, it is assumed to be a servo arm position in degrees. If the value is between 180 and 544, it is also assumed to be in degrees and is truncated to 180. If the value is greater than or equal to 544, it is assumed to be a duty cycle in nanoseconds.
```
Comment (fivdi) [inline README.md]: s/nanoseconds/microseconds/

Comment (nebrius) [reply README.md]: Good catch, will fix once I’m back at laptop later

Comment (nebrius) [reply README.md]: Fixed

**Comment by nebrius** [comment]
@rwaldron @dtex @fivdi ping, any updates for this PR? Are we ready to land it yet?

**Comment by dtex** [comment]
I'm still good with it, but don't have the ability to merge.

**Comment by rwaldron** [comment]
Eek. Sorry about that

**Pull request merged**